src/api/modules/lessons.js

```diff
@@ -83,6 +83,35 @@ function get (tutorial, lessonId) {
  * @example
  * api.lessons.create(tutorial, { title: 'Lesson title', type: 'text' })
  */
+
+function updateQuiz (tutorial, lesson, data) {
```

Comment:
We need to move this to before the doc comment block of the create function.
we need to add more docs comments to each in the future, but for now lets just make sure the create function comments are right before the create function.